src/boss/cloudvm/azure.go

```
@@ -112,6 +114,14 @@ func randomString() string {
 	return strconv.Itoa(r.Int())
 }
 
+func AzureCreateVM(worker *Worker) (*AzureConfig, error) {
+	subscriptionId = os.Getenv("AZURE_SUBSCRIPTION_ID")
+	if len(subscriptionId) == 0 {
+		log.Fatal("AZURE_SUBSCRIPTION_ID is not set.")
```

Why not return an error instead?